packages/support/lib/fs.js

```diff
@@ -16,14 +18,15 @@ import log from './logger';
 import Timer from './timing';
 import pkgDir from 'pkg-dir';
 
-const mkdirAsync = B.promisify(_fs.mkdir);
-const ncpAsync = B.promisify(ncp);
+const mkdirAsync = /** @type {(filepath: string, opts: _fs.MakeDirectoryOptions|undefined) => B<void>} */(B.promisify(_fs.mkdir));
+const ncpAsync = /** @type {(source: string, dest: string, opts: ncp.Options|undefined) => B<void>} */(B.promisify(ncp));
 const findRootCached = _.memoize(pkgDir.sync);
 
 const fs = {
   async hasAccess (path) {
     try {
-      await fs.access(path, _fs.R_OK);
+      // as of recent-ish node versions, `R_OK` moved into the `constants` property of `fs`
+      await fs.access(path, _fs.constants.R_OK);
```

Comment:
shall we use `_fs.constants.R_OK ?? _fs.R_OK` ?

Reply:
I guess the comment is rather ambiguous.  I'll fix that, but the changed one is correct

Reply:
I just removed the comment.  `fs.R_OK` is deprecated, afaict.  guess I don't need to add a comment to explain code that no longer exists, huh